### ISSUE TITLE
No colors

### ISSUE THREAD

**Issue by alexherbo2** [aaronjanse/3mux#75]
- Terminal: [Alacritty]
- OS: [NixOS]

[Alacritty]: https://github.com/alacritty/alacritty
[NixOS]: https://nixos.org

### Installation

``` sh
nix-shell --packages autoreconfHook pkg-config cmake ncurses go libtsm --run '
  GOPATH= go get -u github.com/aaronjanse/3mux
'
```

**Comment by segmentationfaulter** [comment]
same is the case with ubuntu 20.04, terminal is Alacrity and tilix.

**Comment by aaronjanse** [comment]
I'm having trouble reproducing this, but I'm sure this bug exists.

Is there no color at all? As a base case, does the following have color inside and/or outside 3mux?
```
$ echo '\x1b[91mHello \x1b[93mworld!\x1b[m'
```

Thanks!

**Comment by aaronjanse** [comment]
Fixed by #94! :tada:

It was an issue where 3mux recognized multiple styles (e.g. foreground & background) being set at once but accidentally ignore this. Thank you for the bug reports @alexherbo2 and @segmentationfaulter!
